scripts/utils/chain_model.py

```diff
@@ -1,17 +1,32 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from typing import List, Union, Callable, TypeVar, Tuple
```

Comment:
there is no usage for Union
```suggestion
from typing import List, Callable, TypeVar, Tuple
```